src/Tasks/Microsoft.WinFx.targets

```diff
@@ -9,6 +9,6 @@
         the shim will internally simply redirect to the real copy of the targets file in the .NET Framework. 
    -->
 
-   <Import Project="$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets" Condition="Exists('$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets')" />
+   <Import Project="$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets" Condition="Exists('$(MSBuildFrameworkToolsPath)\Microsoft.WinFx.targets') and '$(ImportWinFXTargets)' != 'false'" />
```

Comment:
Check the property first to avoid doing the IO existence check. The and operator is short circuiting.

Reply:
Good call, done!